{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "10-May-2023 12:53:34 DEBUG utils - Directory dataDownload already exists. Skipping.\n",
      "10-May-2023 12:53:34 INFO GEOparse - File already exist: using local version.\n",
      "10-May-2023 12:53:34 INFO GEOparse - Parsing dataDownload/GSE160306_family.soft.gz: \n",
      "10-May-2023 12:53:34 DEBUG GEOparse - DATABASE: GeoMiame\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SERIES: GSE160306\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - PLATFORM: GPL20301\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871112\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871113\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871114\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871115\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871116\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871117\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871118\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871119\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871120\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871121\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871122\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871123\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871124\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871125\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871126\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871127\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871128\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871129\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871130\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871131\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871132\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871133\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871134\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871135\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871136\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871137\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871138\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871139\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871140\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871141\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871142\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871143\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871144\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871145\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871146\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871147\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871148\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871149\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871150\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871151\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871152\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871153\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871154\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871155\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871156\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871157\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871158\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871159\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871160\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871161\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871162\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871163\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871164\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871165\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871166\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871167\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871168\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871169\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871170\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871171\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871172\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871173\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871174\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871175\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871176\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871177\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871178\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871179\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871180\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871181\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871182\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871183\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871184\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871185\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871186\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871187\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871188\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871189\n",
      "10-May-2023 12:53:34 DEBUG GEOparse - SAMPLE: GSM4871190\n"
     ]
    }
   ],
   "source": [
    "import GEOparse\n",
    "import pandas as pd\n",
    "\n",
    "gse = GEOparse.get_GEO(geo=\"GSE160306\", destdir=\"dataDownload\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GPL check:\n",
      "Name:  GPL20301\n",
      "Metadata:\n",
      " - title : Illumina HiSeq 4000 (Homo sapiens)\n",
      " - geo_accession : GPL20301\n",
      " - status : Public on Jun 09 2015\n",
      " - submission_date : Jun 09 2015\n",
      " - last_update_date : Mar 21 2019\n",
      " - technology : high-throughput sequencing\n",
      " - distribution : virtual\n",
      " - organism : Homo sapiens\n",
      " - taxid : 9606\n",
      " - contact_name : ,,GEO\n",
      " - contact_country : USA\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n"
     ]
    }
   ],
   "source": [
    "print(\"GPL check:\")\n",
    "for gpl_name, gpl in geo.gpls.items():\n",
    "    print(\"Name: \", gpl_name)\n",
    "    print(\"Metadata:\",)\n",
    "    for key, value in gpl.metadata.items():\n",
    "        print(\" - %s : %s\" % (key, \", \".join(value)))\n",
    "    print(\"Table data:\",)\n",
    "    print(gpl.table.head())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GSM check:\n",
      "Name:  GSM4871112\n",
      "Metadata:\n",
      " - title : sample_01\n",
      " - geo_accession : GSM4871112\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_01, samplenumber: 1, donor: 04_014, gender: female, age: 80, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: ASA, levoxyl, verpamil, aldactone, nasonex nasal srapy, vicodin, toprol, mirtazapine, pulmicort puff (glucocorticoid), albuterol inhaler 2 puff, home oxygen, Mg tablets,, comorbidities: hypertension (HTN), congestive heart failure (CHF), asthma, history of Vfib with MI x 2 s/p AICD placement in 1999.  Atrial fib, hypothyroidism, pathology_note: pseudophakia with YAG, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 16.95, buffer_volume_unit: ul, ammount_cells: 118, ammount_cells_unit: ng/ul, rin: 6.2, post_mortem_interval: 285, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581359\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871113\n",
      "Metadata:\n",
      " - title : sample_10\n",
      " - geo_accession : GSM4871113\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_10, samplenumber: 10, donor: 16_044, gender: female, age: 78, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Haldol, lorazepam, melatonin, nystatin, omeprazole, Zofran, oxycodone, comorbidities: metastatic duodenal cancer, chemotherapy, small bowel obstruction, hernia repair, hyponatremia, hypertension hyperlipidemia,, pathology_note: none, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 11.9, buffer_volume_unit: ul, ammount_cells: 168, ammount_cells_unit: ng/ul, rin: 5.7, post_mortem_interval: 283, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581358\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871114\n",
      "Metadata:\n",
      " - title : sample_11\n",
      " - geo_accession : GSM4871114\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_11, samplenumber: 11, donor: 04_014, gender: female, age: 80, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: ASA, levoxyl, verpamil, aldactone, nasonex nasal srapy, vicodin, toprol, mirtazapine, pulmicort puff (glucocorticoid), albuterol inhaler 2 puff, home oxygen, Mg tablets,, comorbidities: hypertension (HTN), congestive heart failure (CHF), asthma, history of Vfib with MI x 2 s/p AICD placement in 1999.  Atrial fib, hypothyroidism, pathology_note: pseudophakia with YAG, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 12.05, buffer_volume_unit: ul, ammount_cells: 166, ammount_cells_unit: ng/ul, rin: 8.6, post_mortem_interval: 285, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581357\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871115\n",
      "Metadata:\n",
      " - title : sample_12\n",
      " - geo_accession : GSM4871115\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_12, samplenumber: 12, donor: 06_056, gender: male, age: 70, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: laspofunginin, posacandozole, neupogen, heparin, coumadin, zosyn, vancomycin, comorbidities: ulcerative colitis, bone marrow failure, rhizopus lung infection, refractory anemia with myelodysplasia, pulmonary embolus, HTN, myelodysplasia, dyslipidemia, pathology_note: pterygium removed, glasses, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OS, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 21.3, buffer_volume_unit: ul, ammount_cells: 94, ammount_cells_unit: ng/ul, rin: 8.1, post_mortem_interval: 279, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581356\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871116\n",
      "Metadata:\n",
      " - title : sample_13\n",
      " - geo_accession : GSM4871116\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_13, samplenumber: 13, donor: 05_044, gender: male, age: 81, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: atenolol, DECADRON (STEROIDS), paroxetine, ASA, Aricept, folic acid, comorbidities: metastatic lung CA, HTN, depression, pathology_note: cataract extraction, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 13.33, buffer_volume_unit: ul, ammount_cells: 150, ammount_cells_unit: ng/ul, rin: 6.2, post_mortem_interval: 331, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581355\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871117\n",
      "Metadata:\n",
      " - title : sample_14\n",
      " - geo_accession : GSM4871117\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_14, samplenumber: 14, donor: 06_025, gender: female, age: 82, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Plavix, demadex, levothyroxine, folic acid, metoprolol, amlodipine, hydralzine, ASA, fosamax, isosorbide, Douvent nebulizer, PULMICORT INHALER,, comorbidities: CAD, s/p MI, angina, CHF, COPD, pulmonary HTN, hypothyroidism, breast CA, osteoprosis/OA, vocal cord paralysis, RA, PUD, vitiligo, HTN, pathology_note: pseudophakia, Fuchâ€™s endo dystrophy, choroidal nevi, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 13.16, buffer_volume_unit: ul, ammount_cells: 152, ammount_cells_unit: ng/ul, rin: 6.5, post_mortem_interval: 305, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581354\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871118\n",
      "Metadata:\n",
      " - title : sample_15\n",
      " - geo_accession : GSM4871118\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_15, samplenumber: 15, donor: 06_043, gender: female, age: 84, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: unknown, comorbidities: ovarian CA, mitral regurg, choesystectomy, tonsillectomy, pathology_note: corneal stromal scar, cataract, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 21.28, buffer_volume_unit: ul, ammount_cells: 94, ammount_cells_unit: ng/ul, rin: 4.7, post_mortem_interval: 325, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581353\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871119\n",
      "Metadata:\n",
      " - title : sample_16\n",
      " - geo_accession : GSM4871119\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_16, samplenumber: 16, donor: 06_055, gender: male, age: 63, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Heparin, lasix, lactulose, protonix, paroxetine, vitamin K, comorbidities: liver cirrhosis, alpha-1-AT deficiency diagnosed 2 yrs ago, UTI, HTN, nephrolithiasis, OSA on CPAP at night, pathology_note: none, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 8.51, buffer_volume_unit: ul, ammount_cells: 235, ammount_cells_unit: ng/ul, rin: 8.1, post_mortem_interval: 343, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581430\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871120\n",
      "Metadata:\n",
      " - title : sample_17\n",
      " - geo_accession : GSM4871120\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_17, samplenumber: 17, donor: 08_049, gender: female, age: 80, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Protonix, methotrexate, prednisone, folic acid, captopril, Fosamax, calcium, comorbidities: failure to thrive, COPD, hypertension arthritis, osteoporosis, coronary artery disease, CHF, fractures, pathology_note: corneal dystrophy, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 13.42, buffer_volume_unit: ul, ammount_cells: 149, ammount_cells_unit: ng/ul, rin: 8.1, post_mortem_interval: 280, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581352\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871121\n",
      "Metadata:\n",
      " - title : sample_18\n",
      " - geo_accession : GSM4871121\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_18, samplenumber: 18, donor: 17_040, gender: female, age: 70, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Tylenol, haldol, ativan, morphine,mylicon, comorbidities: Cholangiocarcinoma, sepsis, diabetes type II, coronary artery disease, diverticulitis, liver disease,, pathology_note: none, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OS, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 17.7, buffer_volume_unit: ul, ammount_cells: 113, ammount_cells_unit: ng/ul, rin: 7.6, post_mortem_interval: 295, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581351\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871122\n",
      "Metadata:\n",
      " - title : sample_19\n",
      " - geo_accession : GSM4871122\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_19, samplenumber: 19, donor: 12_021, gender: male, age: 80, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Levaquin, zosyn, Flomax, hydrochlorothiazide, Compazine, lorazepam, nystatin, warfarin, gabapentin, librax, tarceva,, comorbidities: metastatic adenocarcinoma of lung, lung cancer, pulmonary embolism, Waldenstrom macroglobulinemia, hypertension, familial tremor, gastroesophageal disease, asthma, recurring pneumonia, pathology_note: cataract extraction, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OS, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 7.25, buffer_volume_unit: ul, ammount_cells: 275.7, ammount_cells_unit: ng/ul, rin: 7, post_mortem_interval: 356, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581350\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871123\n",
      "Metadata:\n",
      " - title : sample_02\n",
      " - geo_accession : GSM4871123\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_02, samplenumber: 2, donor: 05_014, gender: male, age: 73, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: isoptin, imdur, prozac, trazodone, ASA, comorbidities: CAD, s/p coronary angioplasties, HTN, COPD, hyperlipidemia, anxiety, depression, s/p appendectomy, s/p hemorrhoidectomy., pathology_note: cataracts, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 21.51, buffer_volume_unit: ul, ammount_cells: 93, ammount_cells_unit: ng/ul, rin: 6.3, post_mortem_interval: 354, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581349\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871124\n",
      "Metadata:\n",
      " - title : sample_20\n",
      " - geo_accession : GSM4871124\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_20, samplenumber: 20, donor: 16_044, gender: female, age: 78, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Haldol, lorazepam, melatonin, nystatin, omeprazole, Zofran, oxycodone, comorbidities: metastatic duodenal cancer, chemotherapy, small bowel obstruction, hernia repair, hyponatremia, hypertension hyperlipidemia,, pathology_note: none, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Periphery, sample_group: Control_Periphery, buffer: Water, buffer_volume: 10.81, buffer_volume_unit: ul, ammount_cells: 185, ammount_cells_unit: ng/ul, rin: 5.5, post_mortem_interval: 283, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581348\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871125\n",
      "Metadata:\n",
      " - title : sample_21\n",
      " - geo_accession : GSM4871125\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_21, samplenumber: 21, donor: 04_031, gender: female, age: 88, duration_diabetis: 10 yrs, diabetes_medication: none, other_medication: atenolol/chlorthalidone, monopril, synthroid, ambien, tylenol .3, amitriptyline, comorbidities: DM, HTN, hypothyroidism, depression CHF, osteoarthritis, pathology_note: cataracts, Fuchs., glaucoma suspect, corneal dystrophy, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 19.42, buffer_volume_unit: ul, ammount_cells: 103, ammount_cells_unit: ng/ul, rin: 5.9, post_mortem_interval: 348, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581347\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871126\n",
      "Metadata:\n",
      " - title : sample_22\n",
      " - geo_accession : GSM4871126\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_22, samplenumber: 22, donor: 05_024, gender: male, age: 84, duration_diabetis: >12 years, diabetes_medication: glyburide, insulin, other_medication: atenolol, lisinopril, comorbidities: pancreatic cancer, dyspnea from pleural effusion, DM, Afib., pathology_note: cataract, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 20.41, buffer_volume_unit: ul, ammount_cells: 98, ammount_cells_unit: ng/ul, rin: 6.6, post_mortem_interval: 360, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581395\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871127\n",
      "Metadata:\n",
      " - title : sample_23\n",
      " - geo_accession : GSM4871127\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_23, samplenumber: 23, donor: 06_013, gender: male, age: 77, duration_diabetis: >25 years, diabetes_medication: Glimepiride, Novolin, other_medication: isosorbide, metoprolol, allopurinol, lisinopril, lasix, ASA, Crestor,  Zetia, sulfasalazine, digoxin, comorbidities: ischemic cardiomyopathy, CAD, aortic stenosis, DM, renal insufficiency, hyperlipidemia, HTN, TIA, spinal stenosis, ulcerative colitis, left ulnar palsy,, pathology_note: none, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 9.22, buffer_volume_unit: ul, ammount_cells: 217, ammount_cells_unit: ng/ul, rin: 8.4, post_mortem_interval: 331, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581394\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871128\n",
      "Metadata:\n",
      " - title : sample_24\n",
      " - geo_accession : GSM4871128\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_24, samplenumber: 24, donor: 08_021, gender: male, age: 80, duration_diabetis: 9 years, diabetes_medication: Insulin (Humalin), other_medication: digoxin, prednisone, coumarin, comorbidities: cholecystectomy, prostrate surgery, pathology_note: PVD, cataracts, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 31.3, buffer_volume_unit: ul, ammount_cells: 64, ammount_cells_unit: ng/ul, rin: 8.5, post_mortem_interval: 293, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581393\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871129\n",
      "Metadata:\n",
      " - title : sample_25\n",
      " - geo_accession : GSM4871129\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_25, samplenumber: 25, donor: 14_090, gender: female, age: 74, duration_diabetis: 7 yrs, diabetes_medication: Glimepiride, metformin, other_medication: prednisone, lisinopril/HCTZ, bisoprolol, comorbidities: paratracheal mass, lupus, autoimmune hepatitis, NIDDM, HTN, high creatinine levels., pathology_note: cataracts, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OS, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 18.35, buffer_volume_unit: ul, ammount_cells: 109, ammount_cells_unit: ng/ul, rin: 8.3, post_mortem_interval: 314, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581392\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871130\n",
      "Metadata:\n",
      " - title : sample_26\n",
      " - geo_accession : GSM4871130\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_26, samplenumber: 26, donor: 07_020, gender: female, age: 84, duration_diabetis: 8 years, diabetes_medication: metformin, other_medication: allopurinol, aspirin, ferrous sulfate,  neurontin, omeprazole, KCl, PreserVision, Restasis, Demadex, Tyvaso inhaler, Zetia, comorbidities: breast CA, CHF, A-fib, HTN, pulmonary artery fibrillation/flutter-unsuccessful R atrial ablation; non ischsmic dilated cardiomyopathy, hysterectomy 1992; bowel obstruction, pathology_note: asteroid hyalosis, choroiodal nevus, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OS, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 18.52, buffer_volume_unit: ul, ammount_cells: 108, ammount_cells_unit: ng/ul, rin: 8.1, post_mortem_interval: 343, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581389\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871131\n",
      "Metadata:\n",
      " - title : sample_27\n",
      " - geo_accession : GSM4871131\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_27, samplenumber: 27, donor: 16_005, gender: female, age: 97, duration_diabetis: >15 years, diabetes_medication: none, other_medication: Plavix, metoprolol,dyrenium, gabapentin, multivitamin, comorbidities: DM, HTN, hypothyroidism, depression CHF, osteoarthritis, pathology_note: Phaco, trabeculectomy, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 44.15, buffer_volume_unit: ul, ammount_cells: 45.3, ammount_cells_unit: ng/ul, rin: 5.4, post_mortem_interval: 348, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581388\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871132\n",
      "Metadata:\n",
      " - title : sample_28\n",
      " - geo_accession : GSM4871132\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_28, samplenumber: 28, donor: 16_025, gender: female, age: 90, duration_diabetis: unknown, diabetes_medication: metformin (1000 mg), other_medication: mapap, comorbidities: paroxysmal a-fib on chronic anticoagulation, mild to moderate mitral regurgitation and stenosis, IDDM, HTN, CAD., pathology_note: hypertensive retinopathy, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OS, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 24.39, buffer_volume_unit: ul, ammount_cells: 82, ammount_cells_unit: ng/ul, rin: 6.2, post_mortem_interval: 350, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581387\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871133\n",
      "Metadata:\n",
      " - title : sample_29\n",
      " - geo_accession : GSM4871133\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_29, samplenumber: 29, donor: 17_020, gender: female, age: 54, duration_diabetis: 7 yrs, diabetes_medication: humalog, glucophage, lantus, other_medication: fentanyl, Haldol, heparin, Ativan, levothyroxin, lisinopril, protonix, pantoprazole, comorbidities: acute chronic kidney injury,acute hepatic encephalopathy, heart-to-heart cirrhosis of liver, hypertension, hypothyroidism, obesity, pneumonia, diabetes mellitus type II, IDDM, pathology_note: none, note_other: A1C= 5.9, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OS, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 11.36, buffer_volume_unit: ul, ammount_cells: 176, ammount_cells_unit: ng/ul, rin: 8.6, post_mortem_interval: 321, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581386\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871134\n",
      "Metadata:\n",
      " - title : sample_03\n",
      " - geo_accession : GSM4871134\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_03, samplenumber: 3, donor: 05_035, gender: female, age: 73, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: levoxyl, atenolol, avapro, dilantin, DDAVP, omepraxole, MVI, ASA, comorbidities: breast CA with brain mets, DI, HTN, hypothyroidism, GRD, pathology_note: cataracts, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 16.26, buffer_volume_unit: ul, ammount_cells: 123, ammount_cells_unit: ng/ul, rin: 4.7, post_mortem_interval: 320, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581391\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871135\n",
      "Metadata:\n",
      " - title : sample_30\n",
      " - geo_accession : GSM4871135\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_30, samplenumber: 30, donor: 17_030, gender: male, age: 86, duration_diabetis: 16 years, diabetes_medication: metformin, other_medication: aspirin, levothyroxine,cloidogrel, comorbidities: none noted, pathology_note: AMD, cataract extraction, PVD, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OS, sample_site: Macula, sample_group: Diabetic_Macula, buffer: Water, buffer_volume: 21.05, buffer_volume_unit: ul, ammount_cells: 95, ammount_cells_unit: ng/ul, rin: 5.9, post_mortem_interval: 322, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581390\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871136\n",
      "Metadata:\n",
      " - title : sample_31\n",
      " - geo_accession : GSM4871136\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_31, samplenumber: 31, donor: 04_031, gender: female, age: 88, duration_diabetis: 10 yrs, diabetes_medication: none, other_medication: atenolol/chlorthalidone, monopril, synthroid, ambien, tylenol .3, amitriptyline, comorbidities: DM, HTN, hypothyroidism, depression CHF, osteoarthritis, pathology_note: cataracts, Fuchs., glaucoma suspect, corneal dystrophy, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 20.83, buffer_volume_unit: ul, ammount_cells: 96, ammount_cells_unit: ng/ul, rin: 6.6, post_mortem_interval: 348, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581371\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871137\n",
      "Metadata:\n",
      " - title : sample_32\n",
      " - geo_accession : GSM4871137\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_32, samplenumber: 32, donor: 05_024, gender: male, age: 84, duration_diabetis: >12 years, diabetes_medication: glyburide, insulin, other_medication: atenolol, lisinopril, comorbidities: pancreatic cancer, dyspnea from pleural effusion, DM, Afib., pathology_note: cataract, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 17.86, buffer_volume_unit: ul, ammount_cells: 112, ammount_cells_unit: ng/ul, rin: 4.8, post_mortem_interval: 360, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581370\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871138\n",
      "Metadata:\n",
      " - title : sample_33\n",
      " - geo_accession : GSM4871138\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_33, samplenumber: 33, donor: 06_013, gender: male, age: 77, duration_diabetis: >25 years, diabetes_medication: Glimepiride, Novolin, other_medication: isosorbide, metoprolol, allopurinol, lisinopril, lasix, ASA, Crestor,  Zetia, sulfasalazine, digoxin, comorbidities: ischemic cardiomyopathy, CAD, aortic stenosis, DM, renal insufficiency, hyperlipidemia, HTN, TIA, spinal stenosis, ulcerative colitis, left ulnar palsy,, pathology_note: none, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 16.81, buffer_volume_unit: ul, ammount_cells: 119, ammount_cells_unit: ng/ul, rin: 8.3, post_mortem_interval: 331, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581369\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871139\n",
      "Metadata:\n",
      " - title : sample_34\n",
      " - geo_accession : GSM4871139\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_34, samplenumber: 34, donor: 14_081, gender: male, age: 77, duration_diabetis: unknown, diabetes_medication: Lantus insulin, Humalog insulin, other_medication: fish oil, lasix, Avapro, isosorbide, metroprolol XL, omeprazole, KCl, Crestor, Vit D, nitroglycerin prn, comorbidities: IDDM type 2, adematous colon polyp, hypercholesterolemia, HTN, A-fib w/ ablation, CAD, back pain, stroke, kidney failure, acid reflux, arthritis, atrial flutter, GERD, cataracts, tobacco use., pathology_note: IOL, drusen, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OS, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 20, buffer_volume_unit: ul, ammount_cells: 100, ammount_cells_unit: ng/ul, rin: 7, post_mortem_interval: 331, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581412\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871140\n",
      "Metadata:\n",
      " - title : sample_35\n",
      " - geo_accession : GSM4871140\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_35, samplenumber: 35, donor: 16_005, gender: female, age: 97, duration_diabetis: >15 years, diabetes_medication: none, other_medication: Plavix, metoprolol,dyrenium, gabapentin, multivitamin, comorbidities: DM, HTN, hypothyroidism, depression CHF, osteoarthritis, pathology_note: Phaco, trabeculectomy, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 25.64, buffer_volume_unit: ul, ammount_cells: 78, ammount_cells_unit: ng/ul, rin: 8.1, post_mortem_interval: 314, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581411\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871141\n",
      "Metadata:\n",
      " - title : sample_36\n",
      " - geo_accession : GSM4871141\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_36, samplenumber: 36, donor: 14_097, gender: female, age: 79, duration_diabetis: >12 yrs, diabetes_medication: metformin, other_medication: allopurinol, aspirin, ferrous sulfate, neurontin, omeprazole, KCl, PreserVision, Restasis, Demadex, Tyvaso inhaler, Zetia, comorbidities: heart failure, atrial flutter, chronic renal insufficiency, pulmonary hypertension, Sjogrenâ€™s syndrome, DM, osteoarthritis, pathology_note: Sjogrenâ€™s, Fuchsâ€™, early ARMD, note_other: A1C between 6 and 7 over the years, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 20.62, buffer_volume_unit: ul, ammount_cells: 97, ammount_cells_unit: ng/ul, rin: 9.2, post_mortem_interval: 356, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581410\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871142\n",
      "Metadata:\n",
      " - title : sample_37\n",
      " - geo_accession : GSM4871142\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_37, samplenumber: 37, donor: 17_020, gender: female, age: 54, duration_diabetis: 7 yrs, diabetes_medication: humalog, glucophage, lantus, other_medication: fentanyl, Haldol, heparin, Ativan, levothyroxin, lisinopril, protonix, pantoprazole, comorbidities: acute chronic kidney injury,acute hepatic encephalopathy, heart-to-heart cirrhosis of liver, hypertension, hypothyroidism, obesity, pneumonia, diabetes mellitus type II, IDDM, pathology_note: none, note_other: A1C= 5.9, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OS, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 12.74, buffer_volume_unit: ul, ammount_cells: 157, ammount_cells_unit: ng/ul, rin: 8.2, post_mortem_interval: 348, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581409\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871143\n",
      "Metadata:\n",
      " - title : sample_38\n",
      " - geo_accession : GSM4871143\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_38, samplenumber: 38, donor: 16_025, gender: female, age: 90, duration_diabetis: unknown, diabetes_medication: metformin (1000 mg), other_medication: mapap, comorbidities: paroxysmal a-fib on chronic anticoagulation, mild to moderate mitral regurgitation and stenosis, IDDM, HTN, CAD., pathology_note: hypertensive retinopathy, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 32.3, buffer_volume_unit: ul, ammount_cells: 62, ammount_cells_unit: ng/ul, rin: 8.6, post_mortem_interval: 350, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581346\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871144\n",
      "Metadata:\n",
      " - title : sample_39\n",
      " - geo_accession : GSM4871144\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_39, samplenumber: 39, donor: 16_005, gender: female, age: 97, duration_diabetis: >15 years, diabetes_medication: none, other_medication: Plavix, metoprolol,dyrenium, gabapentin, multivitamin, comorbidities: DM, HTN, hypothyroidism, depression CHF, osteoarthritis, pathology_note: Phaco, trabeculectomy, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OD, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 13.07, buffer_volume_unit: ul, ammount_cells: 153, ammount_cells_unit: ng/ul, rin: 7.8, post_mortem_interval: 321, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581345\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871145\n",
      "Metadata:\n",
      " - title : sample_04\n",
      " - geo_accession : GSM4871145\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_04, samplenumber: 4, donor: 05_044, gender: male, age: 81, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: atenolol, DECADRON (STEROIDS), paroxetine, ASA, Aricept, folic acid, comorbidities: metastatic lung CA, HTN, depression, pathology_note: cataract extraction, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 19.23, buffer_volume_unit: ul, ammount_cells: 104, ammount_cells_unit: ng/ul, rin: 7.4, post_mortem_interval: 331, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581428\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871146\n",
      "Metadata:\n",
      " - title : sample_40\n",
      " - geo_accession : GSM4871146\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_40, samplenumber: 40, donor: 17_030, gender: male, age: 86, duration_diabetis: 16 years, diabetes_medication: metformin, other_medication: aspirin, levothyroxine,cloidogrel, comorbidities: none noted, pathology_note: AMD, cataract extraction, PVD, dr_severity_score: 10, disease_group: Diabetic, dme: FALSE, disease_group_dme: Diabetic, disease_group_detailed: Diabetic, eye: OS, sample_site: Periphery, sample_group: Diabetic_Periphery, buffer: Water, buffer_volume: 16.67, buffer_volume_unit: ul, ammount_cells: 120, ammount_cells_unit: ng/ul, rin: 7, post_mortem_interval: 322, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581344\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871147\n",
      "Metadata:\n",
      " - title : sample_41\n",
      " - geo_accession : GSM4871147\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_41, samplenumber: 41, donor: 06_016, gender: female, age: 65, duration_diabetis: unkown, diabetes_medication: insulin (lantus, humalog), other_medication: allopurinol, ASA, lipitor, plavix, lanoxin, furosemide, indomethacin,  isosorbide, synthroid, toprol, nitroclycerin, altace, comorbidities: CAD, ischemic cardiomyopathy, chronic kidney dz, HTN, anemia, PVD, hypothyroidism, DM type 2, hyperlipidemia, asthma., pathology_note: iris neovascularization, dr_severity_score: 53, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Macula, sample_group: NPDR_Macula, buffer: Water, buffer_volume: 7.4, buffer_volume_unit: ul, ammount_cells: 270, ammount_cells_unit: ng/ul, rin: 6.9, post_mortem_interval: 331, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581343\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871148\n",
      "Metadata:\n",
      " - title : sample_42\n",
      " - geo_accession : GSM4871148\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_42, samplenumber: 42, donor: 08_027, gender: male, age: 84, duration_diabetis: >12 years, diabetes_medication: glipizide, pioglitazone, other_medication: Fosamax, aspirin, atorvastatin, calcium, cefaclor, citalopram, furosemide, gabapentin,  insulin, levothyroxine, mitazapine, omeprazole, polyethilene glycol, repaglinide, triamcinolone, comorbidities: Diabetes mellitus type II, hypothyroidism, hyperlipidemia, pathology_note: Glaucoma suspect, Cataract extraction, dr_severity_score: 35, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Macula, sample_group: NPDR_Macula, buffer: Water, buffer_volume: 32.26, buffer_volume_unit: ul, ammount_cells: 62, ammount_cells_unit: ng/ul, rin: 7.4, post_mortem_interval: 355, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581342\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871149\n",
      "Metadata:\n",
      " - title : sample_43\n",
      " - geo_accession : GSM4871149\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_43, samplenumber: 43, donor: 09_023, gender: male, age: 89, duration_diabetis: >25 yrs, diabetes_medication: Insulin, other_medication: allopurinol, amlopidine, asperin, atenolol, gabapentin, omeprazole, furosemide, lorazepam, comorbidities: DM type 2, IDDM, chronic kidney disease, HTN, prostate adeno CA, melanoma, gout, hyperlipidemia, hypercalcemia., pathology_note: cateract extraction, note_other: Toe amputation 1 yr prior to death, dr_severity_score: 20, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Macula, sample_group: NPDR_Macula, buffer: Water, buffer_volume: 28.57, buffer_volume_unit: ul, ammount_cells: 70, ammount_cells_unit: ng/ul, rin: 8.2, post_mortem_interval: 321, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581341\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871150\n",
      "Metadata:\n",
      " - title : sample_44\n",
      " - geo_accession : GSM4871150\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_44, samplenumber: 44, donor: 09_024, gender: male, age: 69, duration_diabetis: 13 yrs, diabetes_medication: Insulin, other_medication: Furosemide, spironolactone, methotrexate, aranesp, comorbidities: HTN, ascites of unclear etiology, psoriasis, ostheoarthritis., pathology_note: macular pucker, dr_severity_score: 35, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Macula, sample_group: NPDR_Macula, buffer: Water, buffer_volume: 11.56, buffer_volume_unit: ul, ammount_cells: 173, ammount_cells_unit: ng/ul, rin: 7.9, post_mortem_interval: 360, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581340\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871151\n",
      "Metadata:\n",
      " - title : sample_46\n",
      " - geo_accession : GSM4871151\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_46, samplenumber: 46, donor: 15_048, gender: male, age: 97, duration_diabetis: 5 yrs, diabetes_medication: glipizide, other_medication: carvedilol, warfarin, cozaar, digoxin, diltiazem, flector, fluticasone inhalation, furosemide,  januvia, KCl, neurontin, oxycontin, valium, Vicodin, Vit C, Vit E, comorbidities: PVD, chronic left foot ischemia and gangrenous tissue, CHF, suicidal thoughts, chronic severe pain, A-fib, BPH, HTN, chronic back pain, vertigo, constipation, pathology_note: cataract extraction, PVD, dr_severity_score: 20, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Macula, sample_group: NPDR_Macula, buffer: Water, buffer_volume: 9.57, buffer_volume_unit: ul, ammount_cells: 209, ammount_cells_unit: ng/ul, rin: 7.3, post_mortem_interval: 348, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581339\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871152\n",
      "Metadata:\n",
      " - title : sample_47\n",
      " - geo_accession : GSM4871152\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_47, samplenumber: 47, donor: 15_076, gender: male, age: 81, duration_diabetis: 40 yrs, diabetes_medication: insulin, other_medication: hydralazine, Metalozone, comorbidities: colon cancer with pulmonary adn bladder mets, bladder bleeding, bladder clots and occlusion, heart disease, popliteal aneurysm, HTN, CAD in native artery, chronic systolic CHF, CKD, acute on chronic combined systolic and diastolic heart failure, PE, A-fib, gross hematuria, urinary tract obstruction, hypotension, slow transit constipation., pathology_note: PVD, dr_severity_score: 20, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Macula, sample_group: NPDR_Macula, buffer: Water, buffer_volume: 15.5, buffer_volume_unit: ul, ammount_cells: 129, ammount_cells_unit: ng/ul, rin: 5.4, post_mortem_interval: 358, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581338\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871153\n",
      "Metadata:\n",
      " - title : sample_48\n",
      " - geo_accession : GSM4871153\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_48, samplenumber: 48, donor: 16_063, gender: male, age: 62, duration_diabetis: 19 years, diabetes_medication: insulin, other_medication: clonidine,  fluoxetine, gabapentin, hydrocodone, simvastatin,  omeprazole, lisinopril, cephalexin, tirosint, diltiazem, hydralazine, comorbidities: type I diabetes, hypothyroidism, high cholesterol, hypertension, neuropathy, pathology_note: glaucoma suspect due to large cups, but not frank glaucoma, dr_severity_score: 43, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OS, sample_site: Macula, sample_group: NPDR_Macula, buffer: Water, buffer_volume: 16, buffer_volume_unit: ul, ammount_cells: 125, ammount_cells_unit: ng/ul, rin: 8.7, post_mortem_interval: 329, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581337\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871154\n",
      "Metadata:\n",
      " - title : sample_49\n",
      " - geo_accession : GSM4871154\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_49, samplenumber: 49, donor: 17_093, gender: female, age: 66, duration_diabetis: >15 years, diabetes_medication: None, other_medication: unknown, comorbidities: diabetes type 2, cardiomyopathy peripheral artery disease, hypertension, CAD, osteoarthritis, atrial fibrillation, depression, anxiety, history of gangrene, note_other: This donor had very limited health care, dr_severity_score: 53, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Macula, sample_group: NPDR_Macula, buffer: Water, buffer_volume: 35.1, buffer_volume_unit: ul, ammount_cells: 57, ammount_cells_unit: ng/ul, rin: 8.1, post_mortem_interval: 286, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581336\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871155\n",
      "Metadata:\n",
      " - title : sample_05\n",
      " - geo_accession : GSM4871155\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_05, samplenumber: 5, donor: 06_025, gender: female, age: 82, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Plavix, demadex, levothyroxine, folic acid, metoprolol, amlodipine, hydralzine, ASA, fosamax, isosorbide, Douvent nebulizer, PULMICORT INHALER,, comorbidities: CAD, s/p MI, angina, CHF, COPD, pulmonary HTN, hypothyroidism, breast CA, osteoprosis/OA, vocal cord paralysis, RA, PUD, vitiligo, HTN, pathology_note: pseudophakia, Fuchâ€™s endo dystrophy, choroidal nevi, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 14.93, buffer_volume_unit: ul, ammount_cells: 134, ammount_cells_unit: ng/ul, rin: 8.3, post_mortem_interval: 305, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581335\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871156\n",
      "Metadata:\n",
      " - title : sample_50\n",
      " - geo_accession : GSM4871156\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_50, samplenumber: 50, donor: 17_071, gender: male, age: 90, duration_diabetis: unknown, diabetes_medication: glimepiride, other_medication: atenolol, atorvastatin, Azor, furosemide, comorbidities: type II diabetes mellitus, acute pulmonary edema, renal failure, gastrointestinal hemorrhage,, pathology_note: lattice degeneration, vitreous detachment, mild  ARMD, pseudophakia, dr_severity_score: 53, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Macula, sample_group: NPDR_Macula, buffer: Water, buffer_volume: 14.29, buffer_volume_unit: ul, ammount_cells: 140, ammount_cells_unit: ng/ul, rin: 6.2, post_mortem_interval: 297, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581334\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871157\n",
      "Metadata:\n",
      " - title : sample_51\n",
      " - geo_accession : GSM4871157\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_51, samplenumber: 51, donor: 06_016, gender: female, age: 65, duration_diabetis: unkown, diabetes_medication: insulin (lantus, humalog), other_medication: allopurinol, ASA, lipitor, plavix, lanoxin, furosemide, indomethacin,  isosorbide, synthroid, toprol, nitroclycerin, altace, comorbidities: CAD, ischemic cardiomyopathy, chronic kidney dz, HTN, anemia, PVD, hypothyroidism, DM type 2, hyperlipidemia, asthma., pathology_note: iris neovascularization, dr_severity_score: 53, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 14.71, buffer_volume_unit: ul, ammount_cells: 136, ammount_cells_unit: ng/ul, rin: 7.6, post_mortem_interval: 331, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581333\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871158\n",
      "Metadata:\n",
      " - title : sample_52\n",
      " - geo_accession : GSM4871158\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_52, samplenumber: 52, donor: 08_027, gender: male, age: 84, duration_diabetis: >12 years, diabetes_medication: glipizide, pioglitazone, other_medication: Fosamax, aspirin, atorvastatin, calcium, cefaclor, citalopram, furosemide, gabapentin,  insulin, levothyroxine, mitazapine, omeprazole, polyethilene glycol, repaglinide, triamcinolone, comorbidities: Diabetes mellitus type II, hypothyroidism, hyperlipidemia, pathology_note: Glaucoma suspect, Cataract extraction, dr_severity_score: 35, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 21.05, buffer_volume_unit: ul, ammount_cells: 95, ammount_cells_unit: ng/ul, rin: 8.2, post_mortem_interval: 355, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581332\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871159\n",
      "Metadata:\n",
      " - title : sample_53\n",
      " - geo_accession : GSM4871159\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_53, samplenumber: 53, donor: 09_023, gender: male, age: 89, duration_diabetis: >25 yrs, diabetes_medication: Insulin, other_medication: allopurinol, amlopidine, asperin, atenolol, gabapentin, omeprazole, furosemide, lorazepam, comorbidities: DM type 2, IDDM, chronic kidney disease, HTN, prostate adeno CA, melanoma, gout, hyperlipidemia, hypercalcemia., pathology_note: cateract extraction, note_other: Toe amputation 1 yr prior to death, dr_severity_score: 20, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 32.26, buffer_volume_unit: ul, ammount_cells: 62, ammount_cells_unit: ng/ul, rin: 9.1, post_mortem_interval: 321, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581331\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871160\n",
      "Metadata:\n",
      " - title : sample_54\n",
      " - geo_accession : GSM4871160\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_54, samplenumber: 54, donor: 09_024, gender: male, age: 69, duration_diabetis: 13 yrs, diabetes_medication: Insulin, other_medication: Furosemide, spironolactone, methotrexate, aranesp, comorbidities: HTN, ascites of unclear etiology, psoriasis, ostheoarthritis., pathology_note: macular pucker, dr_severity_score: 35, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OS, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 32.79, buffer_volume_unit: ul, ammount_cells: 61, ammount_cells_unit: ng/ul, rin: 7, post_mortem_interval: 360, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581330\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871161\n",
      "Metadata:\n",
      " - title : sample_55\n",
      " - geo_accession : GSM4871161\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_55, samplenumber: 55, donor: 12_004, gender: male, age: 85, duration_diabetis: 17 yrs, diabetes_medication: glipizide, other_medication: Ranexa, hydralazine, tranexamic acid, Ativan, Lasix, potassium chloride, metoprolol, nitroglycerin, imdur, multivitamin, Colace, Motrin, Haldol, roxanol, Zocor, trazodone, allopurinol, digoxin, torsemide, comorbidities: Ischemic cardiomyopathy, CHF, CAD s/p CABG, hyperlipidemia, renal insufficiency/failure, gout, cerebrovascular disease and carotid bruit, HTN, chronic back discomfort, spinal stenosis, atrial fibrillation, heart block with prior pacemaker implantation, anemia, peripheral vascular disease, ex-smoker, pathology_note: cataract extraction, PVD, note_other: allopurinol, amlopidine, asperin, atenolol, gabapentin, omeprazole, furosemide, lorazepam,, dr_severity_score: 20, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 10.7, buffer_volume_unit: ul, ammount_cells: 187, ammount_cells_unit: ng/ul, rin: 7.5, post_mortem_interval: 326, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581329\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871162\n",
      "Metadata:\n",
      " - title : sample_56\n",
      " - geo_accession : GSM4871162\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_56, samplenumber: 56, donor: 15_048, gender: male, age: 97, duration_diabetis: 5 yrs, diabetes_medication: glipizide, other_medication: carvedilol, warfarin, cozaar, digoxin, diltiazem, flector, fluticasone inhalation, furosemide,  januvia, KCl, neurontin, oxycontin, valium, Vicodin, Vit C, Vit E, comorbidities: PVD, chronic left foot ischemia and gangrenous tissue, CHF, suicidal thoughts, chronic severe pain, A-fib, BPH, HTN, chronic back pain, vertigo, constipation, pathology_note: cataract extraction, PVD, dr_severity_score: 20, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 12.99, buffer_volume_unit: ul, ammount_cells: 154, ammount_cells_unit: ng/ul, rin: 7.1, post_mortem_interval: 348, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581328\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871163\n",
      "Metadata:\n",
      " - title : sample_57\n",
      " - geo_accession : GSM4871163\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_57, samplenumber: 57, donor: 15_076, gender: male, age: 81, duration_diabetis: 40 yrs, diabetes_medication: insulin, other_medication: hydralazine, Metalozone, comorbidities: colon cancer with pulmonary adn bladder mets, bladder bleeding, bladder clots and occlusion, heart disease, popliteal aneurysm, HTN, CAD in native artery, chronic systolic CHF, CKD, acute on chronic combined systolic and diastolic heart failure, PE, A-fib, gross hematuria, urinary tract obstruction, hypotension, slow transit constipation., pathology_note: PVD, dr_severity_score: 20, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 15.5, buffer_volume_unit: ul, ammount_cells: 129, ammount_cells_unit: ng/ul, rin: 7.1, post_mortem_interval: 358, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581327\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871164\n",
      "Metadata:\n",
      " - title : sample_58\n",
      " - geo_accession : GSM4871164\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_58, samplenumber: 58, donor: 16_063, gender: male, age: 62, duration_diabetis: 19 years, diabetes_medication: insulin, other_medication: clonidine,  fluoxetine, gabapentin, hydrocodone, simvastatin,  omeprazole, lisinopril, cephalexin, tirosint, diltiazem, hydralazine, comorbidities: type I diabetes, hypothyroidism, high cholesterol, hypertension, neuropathy, pathology_note: glaucoma suspect due to large cups, but not frank glaucoma, dr_severity_score: 43, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OS, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 27.4, buffer_volume_unit: ul, ammount_cells: 73, ammount_cells_unit: ng/ul, rin: 8.2, post_mortem_interval: 329, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581326\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871165\n",
      "Metadata:\n",
      " - title : sample_59\n",
      " - geo_accession : GSM4871165\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_59, samplenumber: 59, donor: 17_069, gender: male, age: 78, duration_diabetis: >10 years, diabetes_medication: Humalog, Lantus, other_medication: aspirin, fish oil, Lasix, Lipitor, lisinopril, warfarin, metolazone, comorbidities: kidney failure, hypertension, melanoma, heart bypass, pathology_note: pseudophakia, epiretinal membrane, dr_severity_score: 35, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OD, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 20, buffer_volume_unit: ul, ammount_cells: 100, ammount_cells_unit: ng/ul, rin: 6.2, post_mortem_interval: 309, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581368\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871166\n",
      "Metadata:\n",
      " - title : sample_06\n",
      " - geo_accession : GSM4871166\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_06, samplenumber: 6, donor: 06_043, gender: female, age: 84, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: unknown, comorbidities: ovarian CA, mitral regurg, choesystectomy, tonsillectomy, pathology_note: corneal stromal scar, cataract, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OS, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 10.1, buffer_volume_unit: ul, ammount_cells: 198, ammount_cells_unit: ng/ul, rin: 7.8, post_mortem_interval: 325, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581367\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871167\n",
      "Metadata:\n",
      " - title : sample_60\n",
      " - geo_accession : GSM4871167\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_60, samplenumber: 60, donor: 17_071, gender: male, age: 90, duration_diabetis: unknown, diabetes_medication: glimepiride, other_medication: atenolol, atorvastatin, Azor, furosemide, comorbidities: type II diabetes mellitus, acute pulmonary edema, renal failure, gastrointestinal hemorrhage,, pathology_note: lattice degeneration, vitreous detachment, mild  ARMD, pseudophakia, dr_severity_score: 53, disease_group: NPDR, dme: FALSE, disease_group_dme: NPDR, disease_group_detailed: NPDR, eye: OS, sample_site: Periphery, sample_group: NPDR_Periphery, buffer: Water, buffer_volume: 26.7, buffer_volume_unit: ul, ammount_cells: 8.4, ammount_cells_unit: ng/ul, rin: 7.5, post_mortem_interval: 297, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581366\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871168\n",
      "Metadata:\n",
      " - title : sample_61\n",
      " - geo_accession : GSM4871168\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_61, samplenumber: 61, donor: 04_028, gender: male, age: 69, duration_diabetis: >25 years, diabetes_medication: insulin, other_medication: pamelor, lisonopril, furosemide, glyburide, atenolol, paxil, insulin, comorbidities: cellulitis with Staph Aureus sepsis, hypertension, coronary atery disease, anxiety, CHF, pathology_note: focal grid laser to macula, IOL, YAG, PVD, dr_severity_score: 43, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OD, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 15.38, buffer_volume_unit: ul, ammount_cells: 130, ammount_cells_unit: ng/ul, rin: 4.7, post_mortem_interval: 320, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581365\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871169\n",
      "Metadata:\n",
      " - title : sample_62\n",
      " - geo_accession : GSM4871169\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_62, samplenumber: 62, donor: 13_030, gender: female, age: 77, duration_diabetis: >15 years, diabetes_medication: Insulin, glipizide,, other_medication: celebrex, ASA, trazodone, norvascstatin, lipitor, hydorchlorothiazide, comorbidities: hypertension, heart attack, acid reflux, congestive heart failure, COPD, aortic valve stenosis, pathology_note: AION, pseudophakia, dr_severity_score: 43, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OS, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 12.3, buffer_volume_unit: ul, ammount_cells: 163, ammount_cells_unit: ng/ul, rin: 7.3, post_mortem_interval: 149, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581364\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871170\n",
      "Metadata:\n",
      " - title : sample_63\n",
      " - geo_accession : GSM4871170\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_63, samplenumber: 63, donor: 05_011, gender: female, age: 73, duration_diabetis: 34 years, diabetes_medication: insulin, other_medication: oxygen, ASA, synthroid, diatx, allopurinol, tums, iron, BiPAP, vit E, fish oil, benicar, allegra, comorbidities: heart failure, cor pumonale and oxygen support, lower extremity lymphedema, pathology_note: repeated pan retinal photocoagulation, dr_severity_score: 65, disease_group: PDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: PDR + DME, eye: OD, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 22.22, buffer_volume_unit: ul, ammount_cells: 90, ammount_cells_unit: ng/ul, rin: 7.6, post_mortem_interval: 312, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581363\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871171\n",
      "Metadata:\n",
      " - title : sample_64\n",
      " - geo_accession : GSM4871171\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_64, samplenumber: 64, donor: 05_011, gender: female, age: 73, duration_diabetis: 34 years, diabetes_medication: insulin, other_medication: oxygen, ASA, synthroid, diatx, allopurinol, tums, iron, BiPAP, vit E, fish oil, benicar, allegra, comorbidities: heart failure, cor pumonale and oxygen support, lower extremity lymphedema, pathology_note: repeated pan retinal photocoagulation, dr_severity_score: 65, disease_group: PDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: PDR + DME, eye: OS, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 21.74, buffer_volume_unit: ul, ammount_cells: 92, ammount_cells_unit: ng/ul, rin: 4.7, post_mortem_interval: 340, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581362\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871172\n",
      "Metadata:\n",
      " - title : sample_65\n",
      " - geo_accession : GSM4871172\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_65, samplenumber: 65, donor: 05_049, gender: female, age: 76, duration_diabetis: 23 years, DR 7 years, diabetes_medication: insulin, other_medication: Diovan, Norvasc, plavix, lopressor, foltex, furosemide, pyridoxine, iron, calcium carbonate, diltiazem, allegra, protoniox, levaquin, coumadin, flagyl, comorbidities: DM type 1, stroke, Afib, hpertension, renal disease, peripheral neuropathy, ischemic bowl disease, pathology_note: focal macula laser, IOL, note_other: drusen, dr_severity_score: 35, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OD, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 28.57, buffer_volume_unit: ul, ammount_cells: 70, ammount_cells_unit: ng/ul, rin: 7.2, post_mortem_interval: 330, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581361\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871173\n",
      "Metadata:\n",
      " - title : sample_66\n",
      " - geo_accession : GSM4871173\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_66, samplenumber: 66, donor: 05_049, gender: female, age: 76, duration_diabetis: 23 years, DR 7 years, diabetes_medication: insulin, other_medication: Diovan, Norvasc, plavix, lopressor, foltex, furosemide, pyridoxine, iron, calcium carbonate, diltiazem, allegra, protoniox, levaquin, coumadin, flagyl, comorbidities: DM type 1, stroke, Afib, hpertension, renal disease, peripheral neuropathy, ischemic bowl disease, pathology_note: grid focal laser, IOL, note_other: drusen, dr_severity_score: 35, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OS, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 45.45, buffer_volume_unit: ul, ammount_cells: 44, ammount_cells_unit: ng/ul, rin: 7.5, post_mortem_interval: 350, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581360\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871174\n",
      "Metadata:\n",
      " - title : sample_67\n",
      " - geo_accession : GSM4871174\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_67, samplenumber: 67, donor: 10_005, gender: male, age: 59, duration_diabetis: unkown, diabetes_medication: insulin, other_medication: unknown, comorbidities: CAD, CHF, COPD, DM type II, HTN, hyperlipidemia, cor pulmonae, ischemic heart disease, morbid obesity, MI, obstructive sleep apnea, tobacco use, dr_severity_score: 35, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OS, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 34.5, buffer_volume_unit: ul, ammount_cells: 58, ammount_cells_unit: ng/ul, rin: 7.7, post_mortem_interval: 280, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581408\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871175\n",
      "Metadata:\n",
      " - title : sample_68\n",
      " - geo_accession : GSM4871175\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_68, samplenumber: 68, donor: 16_055, gender: female, age: 61, duration_diabetis: >17 years, diabetes_medication: Lantus, Humalog, other_medication: allopurinol, ammonium lactate, atorvastatin, diphenhydramine, fuosemide, metoprolol, comorbidities: endometrial cancer, septic shock, acute renal failure, hypertension, heart failure, a fib, diabetes type II, GERD,tinnitus, obesity,, pathology_note: pan retinal photocoagulation, dr_severity_score: 65, disease_group: PDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: PDR + DME, eye: OS, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 15.87, buffer_volume_unit: ul, ammount_cells: 126, ammount_cells_unit: ng/ul, rin: 5.3, post_mortem_interval: 345, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581407\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871176\n",
      "Metadata:\n",
      " - title : sample_69\n",
      " - geo_accession : GSM4871176\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_69, samplenumber: 69, donor: 17_037, gender: female, age: 82, duration_diabetis: unkown, but >8 years, diabetes_medication: metformin, humalog, lantus, other_medication: Aggrenox, amitriptyline, avapro, furosemide, lipitor, metoprolol,  omeprazole, systane, comorbidities: hypertension, GERD, hyperlipidimia, pathology_note: DME OD>OS, note_other: mild ARMD, dr_severity_score: 43, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OD, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 28.17, buffer_volume_unit: ul, ammount_cells: 71, ammount_cells_unit: ng/ul, rin: 7.9, post_mortem_interval: 316, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581406\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871177\n",
      "Metadata:\n",
      " - title : sample_07\n",
      " - geo_accession : GSM4871177\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_07, samplenumber: 7, donor: 06_055, gender: male, age: 63, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Heparin, lasix, lactulose, protonix, paroxetine, vitamin K, comorbidities: liver cirrhosis, alpha-1-AT deficiency diagnosed 2 yrs ago, UTI, HTN, nephrolithiasis, OSA on CPAP at night, pathology_note: none, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 13.42, buffer_volume_unit: ul, ammount_cells: 149, ammount_cells_unit: ng/ul, rin: 7.8, post_mortem_interval: 343, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581405\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871178\n",
      "Metadata:\n",
      " - title : sample_70\n",
      " - geo_accession : GSM4871178\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_70, samplenumber: 70, donor: 17_037, gender: female, age: 82, duration_diabetis: unkown, but >8 years, diabetes_medication: metformin, humalog, lantus, other_medication: Aggrenox, amitriptyline, avapro, furosemide, lipitor, metoprolol,  omeprazole, systane, comorbidities: hypertension, GERD, hyperlipidimia, pathology_note: DME OD>OS, note_other: mild ARMD, dr_severity_score: 35, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OS, sample_site: Macula, sample_group: DME_Macula, buffer: Water, buffer_volume: 29.41, buffer_volume_unit: ul, ammount_cells: 68, ammount_cells_unit: ng/ul, rin: 7.9, post_mortem_interval: 348, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581404\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871179\n",
      "Metadata:\n",
      " - title : sample_71\n",
      " - geo_accession : GSM4871179\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_71, samplenumber: 71, donor: 13_030, gender: female, age: 77, duration_diabetis: >15 years, diabetes_medication: Insulin, glipizide,, other_medication: celebrex, ASA, trazodone, norvascstatin, lipitor, hydorchlorothiazide, comorbidities: hypertension, heart attack, acid reflux, congestive heart failure, COPD, aortic valve stenosis, dr_severity_score: 43, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OD, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 13.7, buffer_volume_unit: ul, ammount_cells: 146, ammount_cells_unit: ng/ul, rin: 4.7, post_mortem_interval: 174, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581403\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871180\n",
      "Metadata:\n",
      " - title : sample_72\n",
      " - geo_accession : GSM4871180\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_72, samplenumber: 72, donor: 04_028, gender: male, age: 69, duration_diabetis: >25 years, diabetes_medication: insulin, other_medication: pamelor, lisonopril, furosemide, glyburide, atenolol, paxil, insulin, comorbidities: cellulitis with Staph Aureus sepsis, hypertension, coronary atery disease, anxiety, CHF, pathology_note: focal grid laser to macula, IOL, YAG, PVD, dr_severity_score: 43, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OS, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 17.09, buffer_volume_unit: ul, ammount_cells: 117, ammount_cells_unit: ng/ul, rin: 7.6, post_mortem_interval: 295, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581402\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871181\n",
      "Metadata:\n",
      " - title : sample_73\n",
      " - geo_accession : GSM4871181\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_73, samplenumber: 73, donor: 05_011, gender: female, age: 73, duration_diabetis: 34 years, diabetes_medication: insulin, other_medication: oxygen, ASA, synthroid, diatx, allopurinol, tums, iron, BiPAP, vit E, fish oil, benicar, allegra, comorbidities: heart failure, cor pumonale and oxygen support, lower extremity lymphedema, pathology_note: repeated pan retinal photocoagulation, dr_severity_score: 65, disease_group: PDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: PDR + DME, eye: OD, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 15.27, buffer_volume_unit: ul, ammount_cells: 131, ammount_cells_unit: ng/ul, rin: 6.7, post_mortem_interval: 312, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581401\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871182\n",
      "Metadata:\n",
      " - title : sample_74\n",
      " - geo_accession : GSM4871182\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_74, samplenumber: 74, donor: 13_030, gender: female, age: 77, duration_diabetis: >15 years, diabetes_medication: Insulin, glipizide,, other_medication: celebrex, ASA, trazodone, norvascstatin, lipitor, hydorchlorothiazide, comorbidities: hypertension, heart attack, acid reflux, congestive heart failure, COPD, aortic valve stenosis, pathology_note: AION, pseudophakia, dr_severity_score: 43, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OS, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 27.78, buffer_volume_unit: ul, ammount_cells: 72, ammount_cells_unit: ng/ul, rin: 7.5, post_mortem_interval: 149, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581400\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871183\n",
      "Metadata:\n",
      " - title : sample_75\n",
      " - geo_accession : GSM4871183\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_75, samplenumber: 75, donor: 05_049, gender: female, age: 76, duration_diabetis: 23 years, DR 7 years, diabetes_medication: insulin, other_medication: Diovan, Norvasc, plavix, lopressor, foltex, furosemide, pyridoxine, iron, calcium carbonate, diltiazem, allegra, protoniox, levaquin, coumadin, flagyl, comorbidities: DM type 1, stroke, Afib, hpertension, renal disease, peripheral neuropathy, ischemic bowl disease, pathology_note: focal macula laser, IOL, note_other: drusen, dr_severity_score: 35, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OD, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 21.28, buffer_volume_unit: ul, ammount_cells: 94, ammount_cells_unit: ng/ul, rin: 8.2, post_mortem_interval: 330, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581399\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871184\n",
      "Metadata:\n",
      " - title : sample_76\n",
      " - geo_accession : GSM4871184\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_76, samplenumber: 76, donor: 05_049, gender: female, age: 76, duration_diabetis: 23 years, DR 7 years, diabetes_medication: insulin, other_medication: Diovan, Norvasc, plavix, lopressor, foltex, furosemide, pyridoxine, iron, calcium carbonate, diltiazem, allegra, protoniox, levaquin, coumadin, flagyl, comorbidities: DM type 1, stroke, Afib, hpertension, renal disease, peripheral neuropathy, ischemic bowl disease, pathology_note: grid focal laser, IOL, note_other: drusen, dr_severity_score: 35, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OS, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 23.26, buffer_volume_unit: ul, ammount_cells: 86, ammount_cells_unit: ng/ul, rin: 8, post_mortem_interval: 350, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581398\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871185\n",
      "Metadata:\n",
      " - title : sample_77\n",
      " - geo_accession : GSM4871185\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_77, samplenumber: 77, donor: 13_059, gender: female, age: 84, duration_diabetis: 7 years, diabetes_medication: glipizide, other_medication: furosemide, verapamil, xopenex, crestor, nexium, venlafaxine, prednisone, hydralazine, azithromysin, spriva, levothyroxine, comorbidities: pulmonary hypertension, hypothyroidism, diabetes type II, reflux disease, COPD, mastectomy due to breast cancer, knee replacement, Gravesâ€™ disease, depression, pathology_note: glaucoma suspect, dr_severity_score: 35, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OD, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 29.41, buffer_volume_unit: ul, ammount_cells: 68, ammount_cells_unit: ng/ul, rin: 4, post_mortem_interval: 230, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581397\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871186\n",
      "Metadata:\n",
      " - title : sample_78\n",
      " - geo_accession : GSM4871186\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_78, samplenumber: 78, donor: 16_055, gender: female, age: 61, duration_diabetis: >17 years, diabetes_medication: Lantus, Humalog, other_medication: allopurinol, ammonium lactate, atorvastatin, diphenhydramine, fuosemide, metoprolol, comorbidities: endometrial cancer, septic shock, acute renal failure, hypertension, heart failure, a fib, diabetes type II, GERD,tinnitus, obesity,, pathology_note: pan retinal photocoagulation, dr_severity_score: 65, disease_group: PDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: PDR + DME, eye: OS, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 25.32, buffer_volume_unit: ul, ammount_cells: 79, ammount_cells_unit: ng/ul, rin: 7.4, post_mortem_interval: 345, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581396\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871187\n",
      "Metadata:\n",
      " - title : sample_79\n",
      " - geo_accession : GSM4871187\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_79, samplenumber: 79, donor: 17_037, gender: female, age: 82, duration_diabetis: unkown, but >8 years, diabetes_medication: metformin, humalog, lantus, other_medication: Aggrenox, amitriptyline, avapro, furosemide, lipitor, metoprolol,  omeprazole, systane, comorbidities: hypertension, GERD, hyperlipidimia, pathology_note: DME OD>OS, note_other: mild ARMD, dr_severity_score: 43, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OD, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 32.79, buffer_volume_unit: ul, ammount_cells: 61, ammount_cells_unit: ng/ul, rin: 7.3, post_mortem_interval: 316, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581416\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871188\n",
      "Metadata:\n",
      " - title : sample_08\n",
      " - geo_accession : GSM4871188\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_08, samplenumber: 8, donor: 08_049, gender: female, age: 80, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: Protonix, methotrexate, prednisone, folic acid, captopril, Fosamax, calcium, comorbidities: failure to thrive, COPD, hypertension arthritis, osteoporosis, coronary artery disease, CHF, fractures, pathology_note: corneal dystrophy, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OD, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 21.05, buffer_volume_unit: ul, ammount_cells: 95, ammount_cells_unit: ng/ul, rin: 7.8, post_mortem_interval: 280, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581415\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871189\n",
      "Metadata:\n",
      " - title : sample_80\n",
      " - geo_accession : GSM4871189\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Periphery\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Periphery, sampleID: sample_80, samplenumber: 80, donor: 17_037, gender: female, age: 82, duration_diabetis: unkown, but >8 years, diabetes_medication: metformin, humalog, lantus, other_medication: Aggrenox, amitriptyline, avapro, furosemide, lipitor, metoprolol,  omeprazole, systane, comorbidities: hypertension, GERD, hyperlipidimia, pathology_note: DME OD>OS, note_other: mild ARMD, dr_severity_score: 35, disease_group: NPDR, dme: TRUE, disease_group_dme: NPDR/PDR + DME, disease_group_detailed: NPDR + DME, eye: OS, sample_site: Periphery, sample_group: DME_Periphery, buffer: Water, buffer_volume: 20.41, buffer_volume_unit: ul, ammount_cells: 98, ammount_cells_unit: ng/ul, rin: 7.9, post_mortem_interval: 348, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581414\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "Name:  GSM4871190\n",
      "Metadata:\n",
      " - title : sample_09\n",
      " - geo_accession : GSM4871190\n",
      " - status : Public on Apr 19 2021\n",
      " - submission_date : Oct 28 2020\n",
      " - last_update_date : Apr 19 2021\n",
      " - type : SRA\n",
      " - channel_count : 1\n",
      " - source_name_ch1 : Retina Macula\n",
      " - organism_ch1 : Homo sapiens\n",
      " - taxid_ch1 : 9606\n",
      " - characteristics_ch1 : tissue: Retina Macula, sampleID: sample_09, samplenumber: 9, donor: 06_056, gender: male, age: 70, duration_diabetis: n/a, diabetes_medication: n/a, other_medication: laspofunginin, posacandozole, neupogen, heparin, coumadin, zosyn, vancomycin, comorbidities: ulcerative colitis, bone marrow failure, rhizopus lung infection, refractory anemia with myelodysplasia, pulmonary embolus, HTN, myelodysplasia, dyslipidemia, pathology_note: pterygium removed, glasses, dr_severity_score: 0, disease_group: Control, dme: FALSE, disease_group_dme: Control, disease_group_detailed: Control, eye: OS, sample_site: Macula, sample_group: Control_Macula, buffer: Water, buffer_volume: 20.41, buffer_volume_unit: ul, ammount_cells: 98, ammount_cells_unit: ng/ul, rin: 5, post_mortem_interval: 279, post_mortem_interval_unit: min\n",
      " - molecule_ch1 : total RNA\n",
      " - extract_protocol_ch1 : Post-mortem human eyes from 43 donors were obtained through the Iowa Lions Eye Bank (Coralville, IA) after written informed consent of the next-of-kin was obtained and preserved within 6 hours post-mortem. Donors were categorized into one of four disease groups: a) diabetic donors with no apparent visual impairment or visible pathology of the retina at last eye exam, b) donors diagnosed with non-proliferative diabetic retinopathy (NPDR) but without diabetic macular edema (DME), c) patients diagnosed with NPDR with additional indication of DME, or d) donors diagnosed with proliferative diabetic retinopathy (PDR) and DME. The diagnosis was based on the early treatment diabetic retinopathy study Diabetic Retinopathy Severity Score (ETDRS-DRSS). Additional retinal samples were taken from healthy donors to serve as the control group. The study was designed such that donors of all groups approximately matched in age and gender distribution. Note that none of the donors received treatment for their retinal indication. The study design was reviewed by the University of Iowa Institutional Review Board and found not to involve human subjects research. From each donor group (healthy control, diabetic, NPPR, and NPDR/PDR + DME), 10 tissue samples were collected from the macula region as well as the retinal periphery. All tissue samples were taken with a 6mm biopsy punch, snap frozen in liquid nitrogen, and preserved at -80C until use. Both miRNA and total RNA was extracted using miRNeasy kits (Qiagen) after homogenization of the retina with Qiashredders (Qiagen). RNA was eluted into 30 μl of dH2O. RNA was quantitatively and qualitatively assessed using the fluorescence-based Broad Range Quant-iT RNA Assay Kit (ThermoFisher) and the Standard Sensitivity RNA Analysis DNF-471 Kit on a 48-channel Fragment Analyzer (Agilent), respectively. Concentrations averaged at 100ng/µL while RIN ranged from 4 to 9., Retina-derived RNA samples were normalized on the MicroLab STAR automated liquid platform (Hamilton). Total RNA input of 240ng was used for library construction with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina #E7760, together with the NEBNext rRNA depletion Kit #E6310 upstream and the NEXNext Multiplex Oligos for Illumina #E7600 downstream (all New England Biolabs). The only deviation from the manufacturer’s protocol was the use of Ampure XP beads (Beckman Coulter) at the double-stranded cDNA purification step, instead of the recommended SPRIselect Beads. Final sequencing libraries were quantified by the fluorescence dye-based methodology High Sensitivity dsDNA Quanti-iT Assay Kit (ThermoFisher) on a Synergy HTX (BioTek). Total RNA libraries were also assessed for size distribution and adapter dimer presence by the High Sensitivity NGS Fragment DNF-474 Kit on a 48-channel Fragment Analyzer (Agilent). Sequencing libraries were normalized on the MicroLab STAR (Hamilton), pooled as two 40-plex pools and spiked in with PhiX Control v3 (Illumina). The two individual pools were then separately clustered on a cBot instrument with a HiSeq 3000/4000 PE Cluster Kit and subsequently sequenced on a HiSeq 4000 Sequencer (all Illumina) with dual index, paired-ends reads at 75 bp length (Read parameters: Rd1: 76, Rd2: 8, Rd3: 8, Rd4: 76) to reach a minimum depth of 50 million Pass-filter reads per sample.\n",
      " - data_processing : Filtered reads were mapped against the Homo sapiens (human) genome hg38/GRCh38 (primary assembly, excluding alternate contigs) using the STAR aligner software (STAR version 2.5.2b) allowing for soft clipping of adapter sequences. For quantification of transcript levels the annotation files from Ensembl version 86 were used, which corresponds to GENCODE 25. Samples were quantified with the above annotations, using RSEM (RSEM version 1.3.0) and featureCount (featureCount version 1.5.1). Quality controls were implemented using FastQC (FastQC version 0.11.5), picardmetrics (picardmetrics version 0.2.4), and dupRadar (dupRadar version 1.0.0) at the respective steps., Normalized log2 Counts Per Million (CPM) mRNA expression values were calculated using the voom function provided by the limma R package (limma version 3.42.0)., mRNA were filtered by minimum read counts using the filterByExpr function of the edgeR package (edgeR version 3.26.5) and. In addition, a Gaussian mixture model (mclust version 5.4.5) was applied, fitting two Gaussian distributions to mean CPM expression data. mRNA assigned to the cluster with higher mean were retained., Surrogate variables describing a technical confounder effect were estimated using the sva R package (sva version 3.34.0). mRNA expression was corrected for donor age as well as the estimated surrogate variable. In more detail, we designed linear models with mRNA expression of each individual transcript as response, and donor age as well as the estimated surrogate variable as covariates. For each linear model, a t-test was performed on the covariate to assess its statistical significance. In case the p-value for a transcript was below 0.05, the expression values of that transcript were replaced by the residuals of the linear model plus the intercept of the model., Genome_build: hg38/GRCh38, Supplementary_files_format_and_content: human_retina_DR_totalRNA_normalized_cpm.txt - normalized cpm values for each gene\n",
      " - platform_id : GPL20301\n",
      " - contact_name : Kolja,,Becker\n",
      " - contact_email : kolja.becker@boehringer-ingelheim.com\n",
      " - contact_department : Global Computational Biology and Digital Sciences\n",
      " - contact_institute : Boehringer Ingelheim Pharma GmbH & Co. KG\n",
      " - contact_address : Birkendorfer Str. 65\n",
      " - contact_city : Biberach an der Riß\n",
      " - contact_zip/postal_code : 88397\n",
      " - contact_country : Germany\n",
      " - instrument_model : Illumina HiSeq 4000\n",
      " - library_selection : cDNA\n",
      " - library_source : transcriptomic\n",
      " - library_strategy : RNA-Seq\n",
      " - relation : BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN16581413\n",
      " - supplementary_file_1 : NONE\n",
      " - series_id : GSE160306, GSE160310\n",
      " - data_row_count : 0\n",
      "Table data:\n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n"
     ]
    }
   ],
   "source": [
    "print(\"GSM check:\")\n",
    "for gsm_name, gsm in geo.gsms.items():\n",
    "    print(\"Name: \", gsm_name)\n",
    "    print(\"Metadata:\",)\n",
    "    for key, value in gsm.metadata.items():\n",
    "        print(\" - %s : %s\" % (key, \", \".join(value)))\n",
    "    print (\"Table data:\",)\n",
    "    print (gsm.table.head())\n",
    "    print(gsm.columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "ename": "KeyError",
     "evalue": "'ID_REF'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "File \u001b[0;32m~/miniconda3/envs/rna/lib/python3.11/site-packages/pandas/core/indexes/base.py:3652\u001b[0m, in \u001b[0;36mIndex.get_loc\u001b[0;34m(self, key)\u001b[0m\n\u001b[1;32m   3651\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m-> 3652\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_engine\u001b[39m.\u001b[39;49mget_loc(casted_key)\n\u001b[1;32m   3653\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mKeyError\u001b[39;00m \u001b[39mas\u001b[39;00m err:\n",
      "File \u001b[0;32m~/miniconda3/envs/rna/lib/python3.11/site-packages/pandas/_libs/index.pyx:147\u001b[0m, in \u001b[0;36mpandas._libs.index.IndexEngine.get_loc\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/miniconda3/envs/rna/lib/python3.11/site-packages/pandas/_libs/index.pyx:176\u001b[0m, in \u001b[0;36mpandas._libs.index.IndexEngine.get_loc\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32mpandas/_libs/hashtable_class_helper.pxi:7080\u001b[0m, in \u001b[0;36mpandas._libs.hashtable.PyObjectHashTable.get_item\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32mpandas/_libs/hashtable_class_helper.pxi:7088\u001b[0m, in \u001b[0;36mpandas._libs.hashtable.PyObjectHashTable.get_item\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;31mKeyError\u001b[0m: 'ID_REF'",
      "\nThe above exception was the direct cause of the following exception:\n",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[41], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m pivoted_control_samples \u001b[39m=\u001b[39m gse\u001b[39m.\u001b[39;49mpivot_samples(\u001b[39m'\u001b[39;49m\u001b[39mVALUE\u001b[39;49m\u001b[39m'\u001b[39;49m)[controls]\n\u001b[1;32m      2\u001b[0m pivoted_control_samples\u001b[39m.\u001b[39mhead()\n",
      "File \u001b[0;32m~/miniconda3/envs/rna/lib/python3.11/site-packages/GEOparse/GEOTypes.py:852\u001b[0m, in \u001b[0;36mGSE.pivot_samples\u001b[0;34m(self, values, index)\u001b[0m\n\u001b[1;32m    850\u001b[0m     tmp_data[\u001b[39m\"\u001b[39m\u001b[39mname\u001b[39m\u001b[39m\"\u001b[39m] \u001b[39m=\u001b[39m gsm\u001b[39m.\u001b[39mname\n\u001b[1;32m    851\u001b[0m     data\u001b[39m.\u001b[39mappend(tmp_data)\n\u001b[0;32m--> 852\u001b[0m ndf \u001b[39m=\u001b[39m concat(data)\u001b[39m.\u001b[39;49mpivot(index\u001b[39m=\u001b[39;49mindex, values\u001b[39m=\u001b[39;49mvalues, columns\u001b[39m=\u001b[39;49m\u001b[39m\"\u001b[39;49m\u001b[39mname\u001b[39;49m\u001b[39m\"\u001b[39;49m)\n\u001b[1;32m    853\u001b[0m \u001b[39mreturn\u001b[39;00m ndf\n",
      "File \u001b[0;32m~/miniconda3/envs/rna/lib/python3.11/site-packages/pandas/core/frame.py:8424\u001b[0m, in \u001b[0;36mDataFrame.pivot\u001b[0;34m(self, columns, index, values)\u001b[0m\n\u001b[1;32m   8419\u001b[0m \u001b[39m@Substitution\u001b[39m(\u001b[39m\"\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[1;32m   8420\u001b[0m \u001b[39m@Appender\u001b[39m(_shared_docs[\u001b[39m\"\u001b[39m\u001b[39mpivot\u001b[39m\u001b[39m\"\u001b[39m])\n\u001b[1;32m   8421\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mpivot\u001b[39m(\u001b[39mself\u001b[39m, \u001b[39m*\u001b[39m, columns, index\u001b[39m=\u001b[39mlib\u001b[39m.\u001b[39mNoDefault, values\u001b[39m=\u001b[39mlib\u001b[39m.\u001b[39mNoDefault) \u001b[39m-\u001b[39m\u001b[39m>\u001b[39m DataFrame:\n\u001b[1;32m   8422\u001b[0m     \u001b[39mfrom\u001b[39;00m \u001b[39mpandas\u001b[39;00m\u001b[39m.\u001b[39;00m\u001b[39mcore\u001b[39;00m\u001b[39m.\u001b[39;00m\u001b[39mreshape\u001b[39;00m\u001b[39m.\u001b[39;00m\u001b[39mpivot\u001b[39;00m \u001b[39mimport\u001b[39;00m pivot\n\u001b[0;32m-> 8424\u001b[0m     \u001b[39mreturn\u001b[39;00m pivot(\u001b[39mself\u001b[39;49m, index\u001b[39m=\u001b[39;49mindex, columns\u001b[39m=\u001b[39;49mcolumns, values\u001b[39m=\u001b[39;49mvalues)\n",
      "File \u001b[0;32m~/miniconda3/envs/rna/lib/python3.11/site-packages/pandas/core/reshape/pivot.py:540\u001b[0m, in \u001b[0;36mpivot\u001b[0;34m(data, columns, index, values)\u001b[0m\n\u001b[1;32m    538\u001b[0m         index_list \u001b[39m=\u001b[39m [Series(data\u001b[39m.\u001b[39mindex, name\u001b[39m=\u001b[39mdata\u001b[39m.\u001b[39mindex\u001b[39m.\u001b[39mname)]\n\u001b[1;32m    539\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m--> 540\u001b[0m     index_list \u001b[39m=\u001b[39m [data[idx] \u001b[39mfor\u001b[39;49;00m idx \u001b[39min\u001b[39;49;00m com\u001b[39m.\u001b[39;49mconvert_to_list_like(index)]\n\u001b[1;32m    542\u001b[0m data_columns \u001b[39m=\u001b[39m [data[col] \u001b[39mfor\u001b[39;00m col \u001b[39min\u001b[39;00m columns_listlike]\n\u001b[1;32m    543\u001b[0m index_list\u001b[39m.\u001b[39mextend(data_columns)\n",
      "File \u001b[0;32m~/miniconda3/envs/rna/lib/python3.11/site-packages/pandas/core/reshape/pivot.py:540\u001b[0m, in \u001b[0;36m<listcomp>\u001b[0;34m(.0)\u001b[0m\n\u001b[1;32m    538\u001b[0m         index_list \u001b[39m=\u001b[39m [Series(data\u001b[39m.\u001b[39mindex, name\u001b[39m=\u001b[39mdata\u001b[39m.\u001b[39mindex\u001b[39m.\u001b[39mname)]\n\u001b[1;32m    539\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m--> 540\u001b[0m     index_list \u001b[39m=\u001b[39m [data[idx] \u001b[39mfor\u001b[39;00m idx \u001b[39min\u001b[39;00m com\u001b[39m.\u001b[39mconvert_to_list_like(index)]\n\u001b[1;32m    542\u001b[0m data_columns \u001b[39m=\u001b[39m [data[col] \u001b[39mfor\u001b[39;00m col \u001b[39min\u001b[39;00m columns_listlike]\n\u001b[1;32m    543\u001b[0m index_list\u001b[39m.\u001b[39mextend(data_columns)\n",
      "File \u001b[0;32m~/miniconda3/envs/rna/lib/python3.11/site-packages/pandas/core/frame.py:3761\u001b[0m, in \u001b[0;36mDataFrame.__getitem__\u001b[0;34m(self, key)\u001b[0m\n\u001b[1;32m   3759\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mcolumns\u001b[39m.\u001b[39mnlevels \u001b[39m>\u001b[39m \u001b[39m1\u001b[39m:\n\u001b[1;32m   3760\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_getitem_multilevel(key)\n\u001b[0;32m-> 3761\u001b[0m indexer \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mcolumns\u001b[39m.\u001b[39;49mget_loc(key)\n\u001b[1;32m   3762\u001b[0m \u001b[39mif\u001b[39;00m is_integer(indexer):\n\u001b[1;32m   3763\u001b[0m     indexer \u001b[39m=\u001b[39m [indexer]\n",
      "File \u001b[0;32m~/miniconda3/envs/rna/lib/python3.11/site-packages/pandas/core/indexes/base.py:3654\u001b[0m, in \u001b[0;36mIndex.get_loc\u001b[0;34m(self, key)\u001b[0m\n\u001b[1;32m   3652\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_engine\u001b[39m.\u001b[39mget_loc(casted_key)\n\u001b[1;32m   3653\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mKeyError\u001b[39;00m \u001b[39mas\u001b[39;00m err:\n\u001b[0;32m-> 3654\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mKeyError\u001b[39;00m(key) \u001b[39mfrom\u001b[39;00m \u001b[39merr\u001b[39;00m\n\u001b[1;32m   3655\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mTypeError\u001b[39;00m:\n\u001b[1;32m   3656\u001b[0m     \u001b[39m# If we have a listlike key, _check_indexing_error will raise\u001b[39;00m\n\u001b[1;32m   3657\u001b[0m     \u001b[39m#  InvalidIndexError. Otherwise we fall through and re-raise\u001b[39;00m\n\u001b[1;32m   3658\u001b[0m     \u001b[39m#  the TypeError.\u001b[39;00m\n\u001b[1;32m   3659\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_check_indexing_error(key)\n",
      "\u001b[0;31mKeyError\u001b[0m: 'ID_REF'"
     ]
    }
   ],
   "source": [
    "pivoted_control_samples = gse.pivot_samples('VALUE')[controls]\n",
    "pivoted_control_samples.head()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "rna",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.3"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
